STX.REST.RESTFulSense.Clients/Brokers/ErrorBrokers/ErrorBroker.cs

```diff
@@ -0,0 +1,26 @@
+﻿// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Newtonsoft.Json;
+using STX.REST.RESTFulSense.Clients.Models.Brokers.StatusDetails;
+
+namespace STX.REST.RESTFulSense.Clients.Brokers.ErrorBrokers
+{
+    internal partial class ErrorBroker : IErrorBroker
+    {
+        public ErrorBroker() =>
+            statusDetails = InitialiseStatusCodes();
+
+        private static IQueryable<StatusDetail> InitialiseStatusCodes()
+        {
+            string path = Path.Combine(Directory.GetCurrentDirectory(), "Data\\StatusCodes.json");
```

Comment:
I think this can´t be done under broker because has a sequencing flow.
Maybe we need a FileBroker to read the json, then we need a deserialization broker to abstract the serialization piece and orchestrate both.

Reply:
@hassanhabib what is your thoughts here?  Are we allowed to read the and serialise the file here like this since its the same kind of thing we do for loading JSON config on a storage broker  OR would be need 2 brokers and a foundation service to internally orchestrate the read via a file broker and then the deserialization via another broker?

Reply:
@cjdutoit - I agree. @LBoullosa is right. This looks like sequencing. which is logic. We would need two brokers and maybe two Foundation services.